dev-java/protobuf-java/protobuf-java-3.21.5.ebuild

```diff
@@ -0,0 +1,90 @@
+			-> ${PARENT_P}.tar.gz
+	"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86 ~amd64-linux ~x86-linux ~x64-macos"
+fi
+
+DESCRIPTION="Google's Protocol Buffers - Java bindings"
+HOMEPAGE="
+	https://developers.google.com/protocol-buffers/
+	https://github.com/protocolbuffers/protobuf
+"
+
+LICENSE="BSD"
+SLOT="0/32"
+
+S="${WORKDIR}/${PARENT_P}/java"
+
+BDEPEND="
+	dev-libs/${PARENT_PN}:${SLOT}
+	>=virtual/jdk-1.8:*
+"
+
+DEPEND=">=virtual/jdk-1.8:*"
+RDEPEND=">=virtual/jre-1.8:*"
+
+JAVA_SRC_DIR="core/src/main/java"
+JAVA_RESOURCE_DIRS="core/src/main/resources"
+JAVA_AUTOMATIC_MODULE_NAME="com.google.protobuf"
+JAVA_JAR_FILENAME="protobuf.jar"
+
+src_prepare() {
+	pushd "${WORKDIR}/${PARENT_P}" > /dev/null || die
+	default
+	popd > /dev/null || die
+
+	java-pkg-2_src_prepare
+
+	# Copy resources from ../src/google/protobuf according to
+	# https://github.com/protocolbuffers/protobuf/blob/v3.21.2/java/core/pom.xml#L45-L61
+	mkdir -p core/src/main/resources/google/protobuf/compiler || die
+	cp {../src,core/src/main/resources}/google/protobuf/compiler/plugin.proto || die
+	cp ../src/google/protobuf/{any,api,descriptor,duration,empty,field_mask,source_context,struct,timestamp,type,wrappers}.proto \
+		"${JAVA_RESOURCE_DIRS}/google/protobuf" || die
+
+	# Generate 146 .java files according to
+	# https://github.com/protocolbuffers/protobuf/blob/v3.21.5/java/core/generate-sources-build.xml
+	"${BROOT}/usr/bin/protoc" --java_out=core/src/main/java -I../src ../src/google/protobuf/any.proto || die
+	"${BROOT}/usr/bin/protoc" --java_out=core/src/main/java -I../src ../src/google/protobuf/api.proto || die
+	"${BROOT}/usr/bin/protoc" --java_out=core/src/main/java -I../src ../src/google/protobuf/compiler/plugin.proto || die
+	"${BROOT}/usr/bin/protoc" --java_out=core/src/main/java -I../src ../src/google/protobuf/descriptor.proto || die
+	"${BROOT}/usr/bin/protoc" --java_out=core/src/main/java -I../src ../src/google/protobuf/duration.proto || die
+	"${BROOT}/usr/bin/protoc" --java_out=core/src/main/java -I../src ../src/google/protobuf/empty.proto || die
+	"${BROOT}/usr/bin/protoc" --java_out=core/src/main/java -I../src ../src/google/protobuf/field_mask.proto || die
+	"${BROOT}/usr/bin/protoc" --java_out=core/src/main/java -I../src ../src/google/protobuf/source_context.proto || die
+	"${BROOT}/usr/bin/protoc" --java_out=core/src/main/java -I../src ../src/google/protobuf/struct.proto || die
+	"${BROOT}/usr/bin/protoc" --java_out=core/src/main/java -I../src ../src/google/protobuf/timestamp.proto || die
+	"${BROOT}/usr/bin/protoc" --java_out=core/src/main/java -I../src ../src/google/protobuf/type.proto || die
+	"${BROOT}/usr/bin/protoc" --java_out=core/src/main/java -I../src ../src/google/protobuf/wrappers.proto || die
+
+	JAVA_SRC_DIR="core/src/main/java" JAVA_JAR_FILENAME="protobuf.jar" java-pkg-simple_src_compile
+}
+
+src_install() {
+	JAVA_SRC_DIR="core/src/main/java" JAVA_JAR_FILENAME="protobuf.jar" java-pkg-simple_src_install
+}
```

Comment:
Also here adding src_install() appears useless.

Reply:
Same commit than the previous file.